src/SFA.DAS.Apim.Developer.Web/Views/Error/AccessDenied.cshtml

```diff
@@ -0,0 +1,37 @@
+@model string
+@using SFA.DAS.MA.Shared.UI.Models.Links
+@{
+    if (Configuration["AuthType"].Equals("employer", StringComparison.CurrentCultureIgnoreCase))
+    {
+        Layout = "_LayoutEmployer";
+    }
+    
+    ViewData["Title"] = "Get started with the recruitment API";
+}
+<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+
+        <div class="govuk-grid-column-two-thirds">
+
+            <h1 class="govuk-heading-xl">Access denied</h1>
+
+            @if (Configuration["AuthType"].Equals("provider", StringComparison.CurrentCultureIgnoreCase))
+            {
+                <p class="govuk-body">You don’t have permission to access the Apprenticeship Service.</p>
+                <h2 class="govuk-heading-m">Help</h2>
+                <p class="govuk-body">You can contact the Apprenticeship Service for advice or help on how to use the service.</p>
+                <h3 class="govuk-heading-m">Apprenticeship Service</h3>
+                <p class="govuk-body">Telephone: 0800 015 0600 <a href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a></p>
```

Comment:
Can we add `class="govuk-link"` to the link?

Reply:
👍🏻 thats done